dev/service-registry/play-integration-javadsl/src/main/scala/com/lightbend/lagom/play/LagomPlayModule.scala

```diff
@@ -74,9 +77,15 @@ class LagomPlayModule extends Module {
   }
 }
 
-class PlayRegisterWithServiceRegistry @Inject() (config: Configuration, serviceInfo: ServiceInfo, serviceRegistry: ServiceRegistry, applicationLifecycle: ApplicationLifecycle) {
-  private val httpAddress = config.underlying.getString("play.server.http.address")
-  private val httpPort = config.underlying.getString("play.server.http.port")
+class PlayRegisterWithServiceRegistry @Inject() (config: Config, serviceInfo: ServiceInfo, serviceRegistry: ServiceRegistry, applicationLifecycle: ApplicationLifecycle) {
+
+  @deprecated(message = "prefer constructor using typesafe Config instead", since = "1.4.0")
```

Comment:
👍 this is a case where retaining a deprecated constructor is warranted (and maybe required by the build). I wouldn't expect this class to actually be used outside the framework code, but it is technically possible.